docs/preview.md

```diff
@@ -0,0 +1,53 @@
+# Early access to MSTest packages
+
+Stable versions (and selected previews) of MSTest, and related packages, are distributed through [nuget.org](https://www.nuget.org/packages/MSTest)
+
+We also publish every successful merge to main and release branches to our preview nuget channel test-tools.
+
+To use this channel it needs to be added to your configuration, typically by creating [NuGet.Config](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file) file with the following content. And placing it next to your solution file:
```

Comment:
I would word it differently. Something like `To use this channel, you will need to add the following feed .... This is typically done by adding or editing the NuGet.config <link> file next to your solution. Here is a fully fledge example`

Reply:
I wanted to avoid repeating the channel twice in the text. Can you give me full edit so I can see what you mean exactly?

Reply:
Just merge it.

![StarWarsIdentificationGIF](https://github.com/microsoft/testfx/assets/5735905/cac1362b-89e1-4e50-ae5f-4e2ec6a94691)